functional_plant_indicators_seminat.Rmd

```diff
@@ -101,7 +101,7 @@ The ANO evaluation data cover the first three years, 2019-2021, of the first 5-y
 
 ## Reference state and values
 ### Reference state
-The reference state is defined via the functional signature of the generalized species lists in NiN (see also Töpper et al. 2018). By bootstrapping the species lists (see details further below) and calculating community-weighted means of functional plant indicators for every re-sampled community, we describe the reference state as a distribution of indicator values for each respective plant functional indicator. These distributions are calculated for ecosystem sub-types ("grunntyper" or "kartleggingsenheter" at a 1:5000 mapping scale) within the main wetland types in NiN. A more extensive discussion on the use of reference communities can be found in Jakobsson et al. (2020).
+The reference state is defined via the functional signature of the generalized species lists in NiN (see also Töpper et al. 2018). By bootstrapping the species lists (see details further below) and calculating community-weighted means of functional plant indicators for every re-sampled community, we describe the reference state as a distribution of indicator values for each respective plant functional indicator. These distributions are calculated for ecosystem sub-types [?] ("grunntyper" or "kartleggingsenheter" at a 1:5000 mapping scale) within the main wetland types in NiN. A more extensive discussion on the use of reference communities can be found in Jakobsson et al. (2020).
```

Comment:
[?] - Halvorsen et al (2020) translate "grunntype" as "minor type" (and "hovedtype" as "major type"). But i guess "sub-types" is perfectly fine here, since it subsumes grunntyper and kartleggingsenheter. So i guess you can ignore this comment...

Reply:
still, it is better to be consistent with established literature. So, I corrected this for all FPIs in the upcoming pull-request